exercises05/ExercisePPJ_05_1.java

```diff
@@ -0,0 +1,16 @@
+/*
+Dana jest zmienna typu boolean czyPada zainicjowana wartością true.
+Napisz program sprawdzający stan tej zmiennej i wyświetlający stosowny komunikat na ekran (wykorzystaj instrukcję warunkową if).
+*/
+
+public class ExercisePPJ_05_1 {
+	public static void main(String[] args) {
+		boolean czyPada = true;
+		
+		if(czyPada == true) {
```

Comment:
each Boolean value can be true or false. 
so, when we replace `czyPada` variable with it's actual value than expression looks like that:
`if(true == true)`
which is true indeed. But if you get rid of ` == true` it's also a correct if statement:
`if(czyPada)` which evaluates to  `if(true)`

Do you see what am I talking about?:)
You don't need to use Boolean operators, because value you want to evaluate is of type Boolean already:)

Reply:
@MatDls ok I'm understand, enough check if true